engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/EngineJobExecutionActor.scala

```diff
@@ -483,16 +492,15 @@ class EngineJobExecutionActor(replyTo: ActorRef,
 
   private def forwardAndStop(response: BackendJobExecutionResponse): State = {
     replyTo forward response
-    returnExecutionToken()
-    instrumentJobComplete(response)
-    pushExecutionEventsToMetadataService(jobDescriptorKey, eventList)
-    recordExecutionStepTiming(stateName.toString, currentStateDuration)
-    context stop self
```

Comment:
Looks like you're removing a lot more than just the metadata writing?

Reply:
If you look at what was removed, it was removed twice twice and copypastad back once. FTFY'ed the rest of the file while here cleaning up [an IntelliJ warning](https://www.jetbrains.com/help/idea/analyzing-duplicates.html).